src/main/groovy/life/qbic/datamodel/dtos/projectmanagement/ProjectSpace.groovy

```diff
@@ -23,11 +23,21 @@ final class ProjectSpace {
      */
     final String name
 
+    private static final def REGEX = ~'[-\\w]+'
```

Comment:
I don't understand this regular expression. What does it match?

Reply:
Maybe a more declarative variable name would improve readability as well

Reply:
+ matches the previous token between one and unlimited times, as many times as possible, giving back as needed (greedy)
- matches the character - with index 4510 (2D16 or 558) literally (case sensitive)
\w matches any word character (equivalent to [a-zA-Z0-9_])

not sure about the tilde, but that I took from the project code implementation

Reply:
pushed changes